src/switch_ivr_async.c

```diff
@@ -1404,6 +1404,9 @@ static void *SWITCH_THREAD_FUNC recording_thread(switch_thread_t *thread, void *
 				switch_channel_hangup(channel, SWITCH_CAUSE_DESTINATION_OUT_OF_ORDER);
 				switch_core_session_reset(session, SWITCH_TRUE, SWITCH_TRUE);
 			}
+			
+			// We might need to consider a threshold counter that we need to exceed first before we prune it
+			switch_set_flag(bug, SMBF_PRUNE);
```

Comment:
We need to add that channel variable to immediately stop media bug on write fail similar to the statement above which check for `hangup_on_error` e.g. `stop_on_write_fail` or something

Reply:
Flipper based upon chanvar in the code now.  RECORD_STOP_WRITE_ON_ERROR.